app/src/main/java/com/imashnake/animite/features/media/MediaPage.kt

```diff
@@ -46,211 +47,205 @@ fun MediaPage(
     val media = viewModel.uiState
 
     // TODO: [Add shimmer](https://google.github.io/accompanist/placeholder/)
-    Box(
-        Modifier
-            .fillMaxSize()
-            .verticalScroll(rememberScrollState())
-    ) {
-        // TODO: How do I align this?
-        if (!media.bannerImage.isNullOrEmpty()) {
-            Box {
-                AsyncImage(
-                    model = media.bannerImage,
-                    contentDescription = "Banner Image",
-                    contentScale = ContentScale.FillHeight,
-                    modifier = Modifier.height(168.dp),
-                    alignment = Alignment.Center
-                )
+    AnimiteTheme {
```

Comment:
As mentioned above, these are quite heavily nested Composables, I'd like to see them split out a little bit more, so that only the relevant parameters are passed to each smaller Composable function to increase the chance of them being tested in isolation/at least to get a Preview working

Reply:
Added to #48.